content/practices/research-workshop/index.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "Research Workshop"
```

Comment:
Might worth to change it to "User Research Introduction" or "Research Intro & De-risking Assumptions: Exploratory Research"
As the content describe the introduction to user research

Reply:
Updated.